calculator_kuba.py

```diff
@@ -39,15 +42,33 @@ def add_dot():
     current_text = entry.get()
     if '.' not in current_text:
         entry.insert(tk.END , '.')
+        
+def calculation_history():
+    history_window = tk.Toplevel(root)
+    history_window.title("Calculation history:")
+    history_text = tk.Text(history_window)
+    history_text.pack(expand=True, fill=tk.BOTH)
+    for calculation in calculations:
+        history_text.insert(tk.END, f"{calculation[0]} = {calculation[1]}\n")
+        
+def save_to_csv(calculations ,  file_name = 'Calculation history.csv'):    
+    desktop_path = os.path.join(os.path.expanduser('~') , 'Desktop')
```

Comment:
Nie zapisujemy rzeczy na pulpicie. Albo robimy to w directory w katalogu gdzie znajduje się Nasz plik, albo ma wyskakiwać okno, w którym możemy sami sobie wybrać miejsce i wpisać nazwę. Na linuxie by to nie przeszło

Reply:
plik zapisuje się w directory w katalogu gdzie znajduje się nasz plik